OleksandraKotovska/HW5/3.py

```diff
@@ -0,0 +1,21 @@
+customer_number = int(input("Enter you number to calculate its factorial: "))
```

Comment:
good job

look at this code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)